WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldEditorViewModel.swift

```diff
@@ -70,6 +73,15 @@ final class CustomFieldEditorViewModel: ObservableObject {
     }
 }
 
+private extension String {
+    func prettyPrint() -> String {
+        return (try? JSONSerialization.jsonObject(with: self.data(using: .utf8) ?? Data()))
+            .flatMap { try? JSONSerialization.data(withJSONObject: $0, options: .prettyPrinted) }
```

Comment:
I looked into this and it seems like there's a possibility this might re-order the JSON items because JSONSerialization treats JSON objects as dictionaries, which don't guarantee order. I don't think this is an issue that we need to fix, though, as the value is shown read-only anyway.

There's some non-trivial solutions here https://stackoverflow.com/questions/42290866/ordered-dictionary-in-json so I thought to just mention it.

Reply:
Yes, I agree, this is an issue also in Android, and as I said, I don't think it's a major issue, since the JSON would be generally not manually typed, so the order doesn't matter really.